src/test/java/org/springframework/data/couchbase/domain/UserRepository.java

```diff
@@ -30,10 +32,14 @@
  * @author Michael Reiche
  */
 @Repository
-public interface UserRepository extends PagingAndSortingRepository<User, String> {
+public interface UserRepository extends CouchbaseRepository<User, String> {
 
 	List<User> findByFirstname(String firstname);
 
+	List<User> findByFirstnameIn(String... firstnames);
```

Comment:
isn't this related to the other open pr?

Reply:
Yes. Removed

Reply:
I think this line will be back in after rebasing to master that has that change.